cmd/cli/plugin/airgapped-network/cmd/client_imgpkg.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
```

Comment:
`cmd` is a package name that usually contains main.go. Is it possible to give some other package name other than `cmd`?